### PR TITLE
Prints displayName and preinstall at beginning of mods:install

### PR DESCRIPTION
**Pull request by joehan**
Prints displayName and preinstall at beginning of mods:install
Screen Shot 2019-07-30 at 11 45 50 AM

### CODE REVIEW

File: src/commands/mods-install.ts [hunk 1]
```diff
@@ -40,6 +40,8 @@ async function installMod(options: InstallModOptions): Promise<void> {
   const spec = source.spec;
   const spinner = ora.default("Installing your mod instance. This usually takes 3 to 5 minutes...");
   try {
+    logger.info(clc.bold(spec.displayName));
+    logger.info(spec.preinstallContent);
```
Comment (bkendall) [inline src/commands/mods-install.ts]: This is markdown, right? Why aren't we nicely printing the rendered markdown?

Comment (joehan) [reply src/commands/mods-install.ts]: Fixed

**Comment by bkendall** [comment]
I left a question in the internal bug for this - let's discuss there (b/134783324)

**Comment by joehan** [comment]
Replied!

**Comment by laurenzlong** [comment]
Closing this PR since it is not a desirable change. See https://b.corp.google.com/issues/134783324#comment5.

**Comment by milyes** [comment]
> Imprime displayName et preinstall au début de mods:install Capture d&apos;écran 2019-07-30 à 11h45 50

**Pull request closed without merging**